docs/tutorial/mlflow.rst

```diff
@@ -77,7 +77,19 @@ To install Juju from snap, run this command:
 
 .. code-block:: bash
 
-   sudo snap install juju --classic --channel=2.9/stable
+   sudo snap install juju --channel=3.4/stable
+
+On some machines there might be a missing folder which is required for Juju to run correctly. To ensure that this folder exists, run:
+
+.. code-block:: bash
+   
+   mkdir -p ~/.local/share
+
+As a next step, configure MicroK8s to work properly with Juju by running:
+
+.. code-block:: bash
+
+   microk8s config | juju add-k8s my-k8s --client
```

Comment:
Is this really necessary on local machine? Without doing anything I can see microk8s as clouds.
```bash
microk8s   1        localhost  k8s   1            built-in  A Kubernetes Cluster
```

Reply:
I think we should keep this one .. I remember we were having problems in aws machines or somewhere ... we also have this in our kubeflow guide https://charmed-kubeflow.io/docs/get-started-with-charmed-kubeflow